direct/functionals/__init__.py

```diff
@@ -1,5 +1,9 @@
 # coding=utf-8
 # Copyright (c) DIRECT Contributors
+
 from direct.functionals.challenges import *
+from direct.functionals.grad import *
+from direct.functionals.nmae import NMAELoss
+from direct.functionals.nmse import *
```

Comment:
Don't use `*` imports. Unless you define `__all__` in these modules

Reply:
There is an __all__ in all of those